README.md

```diff
@@ -80,9 +88,11 @@ If there are changes made in the Original repository, you can sync the fork to k
 4. ```git merge upstream/master``` to sync your local `master' branch with `upstream/master`. **Note**: Your local changes will be retained and your fork's master branch will be in sync with the upstream repository.
 
 ## Requirements
+
 * [Android Studio](http://developer.android.com/sdk/index.html)
 
 ## Resources
+
 * [The Maps-App](https://github.com/Esri/maps-app-android/blob/master/maps-app/README.md)
```

Comment:
It appears this link is broken, likely because we're referencing the master blob.

Reply:
Good one. Are you okay with removing that line altogether? Seems silly to link _to_ the document _from_ the document.